addon/components/frost-fixed-table.js

```diff
@@ -429,6 +417,28 @@ export default Component.extend({
     }
   },
 
+  setMinimumCellWidths (bodySelector, numColumns, accountForSelection) {
+    for (let pos = 1; pos <= numColumns; ++pos) {
+      const curBodyColumn = this.$(`${bodySelector} .frost-table-row .frost-table-row-cell:nth-child(` +
```

Comment:
Oh ok. Then a selector would be a more succinct way of getting the nth column of every row. I'd be okay if you just stored the `pos` before hand so you didn't need to have to combine your template with a `+`.